internal/api/operations/update.go

```diff
@@ -36,6 +36,15 @@ func (op UpdateOperation) Apply(tx db.RWTx) (*db.QueryResult, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	root = tx.Ref(op.ModelID)
+	clauses = handleFindMany(tx, root, op.FindArgs)
+	q = tx.Query(root, clauses...)
+	final := q.All()
+	if len(final) == 1 {
```

Comment:
same as before.. not sure i understand this..

Reply:
Ditto above